File: .vscode/direccion _ip.py
def validar_octeto(octeto):
    try:
        valor = int(octeto)
        return 0 <= valor <= 255
    except ValueError:
        return False

def validar_direccion_ip(direccion_ip):
    octetos = direccion_ip.split(".")
    if len(octetos) != 4:
        return False
    for octeto in octetos:
        if not validar_octeto(octeto):
            return False
    return True

# Prueba la función con una dirección IP
direccion_ip = "192.168.0.1"
if validar_direccion_ip(direccion_ip):
    print(f"La dirección IP {direccion_ip} es válida.")
else:
    print(f"La dirección IP {direccion_ip} no es válida.")

File: .vscode/direccion ip.js
function validarOcteto(octeto) {
    let valor = parseInt(octeto, 10);
    return valor >= 0 && valor <= 255;
}

function validarDireccionIP(direccionIP) {
    let octetos = direccionIP.split(".");
    if (octetos.length !== 4) {
        return false;
    }
    for (let i = 0; i < octetos.length; i++) {
        if (!validarOcteto(octetos[i])) {
            return false;
        }
    }
    return true;


// Prueba la función con una dirección IP
let direccionIP = "192.168.0.1";
if (validarDireccionIP(direccionIP)) {
    console.log(`La dirección IP ${direccionIP} es válida.`);
} else {
    console.log(`La dirección IP ${direccionIP} no es válida.`);
}
